gui/quantum.lua

```diff
@@ -11,262 +11,218 @@ local quickfort = reqscript('quickfort')
 local quickfort_command = reqscript('internal/quickfort/command')
 local quickfort_orders = reqscript('internal/quickfort/orders')
 
-QuantumUI = defclass(QuantumUI, guidm.MenuOverlay)
+
+-- UI Layout
+QuantumUI = defclass(QuantumUI, gui.ZScreen)
 QuantumUI.ATTRS {
-    frame_inset=1,
     focus_path='quantum',
-    sidebar_mode=df.ui_sidebar_mode.LookAround,
+    pass_movement_keys=true,
+    pass_mouse_clicks=false
 }
 
-function QuantumUI:init()
-    local cart_count = #assign_minecarts.get_free_vehicles()
+QuantumWin = defclass(QuantumWin, widgets.Window)
+QuantumWin.ATTRS {
+    frame_title = 'Quantum Stockpile',
+    frame = { w=60, h=26 },
+    resizable = true,
+    autoarrange_subviews=true
```

Comment:
Could you post a screenshot of the UI? I suspect the dimensions could use some tweaking. also, you might benefit from setting `autoarrange_gap=1`